tests/torch_mlir/torch-mlir-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Torch-MLIR
 -f https://llvm.github.io/torch-mlir/package-index/
-torch-mlir
+torch-mlir==20230616.871
```

Comment:
I've been playing a kind of dumb game not pinning this. It was kind of okay because there's not a hard dependence on the wheel in any way except for tests (which weren't hard fails). So the solution here is to pin this dependency so that if CI fails you can repro locally (i.e., in whatever version that was previously being pip installed there's a segfault/fail).